week-6-benchmarking/README.md

```diff
@@ -0,0 +1,171 @@
+# Benchmarking
+
+## PR benchmark for your model forward pass
+
+Run:
+
+```bash
+make benchmark-forwardpass
+```
+
+Results:
+
+```
+---------------- benchmark: 2 tests ----------------
+Name (time in ms)                     Mean          
+----------------------------------------------------
+test_benchmark_forwardpass_gpu      6.8554 (1.0)    
+test_benchmark_forwardpass_cpu     35.3109 (5.15)   
```

Comment:
let's add environment here,  benchmarks are not complete without env you used to run them

Reply:
Readme updated. In short:

Test set up:
- CPU: Intel Core i5-8300H CPU @ 2.30GHz x 8
- RAM: 15.5GiB
- GPU: NVIDIA GeForce GTX 1050 Ti
- DRAM: 4096 MiB
- CUDA: 11.7
- Noisy neighbours: default minikube cluster activity